src/components/sbb-tooltip/sbb-tooltip.tsx

```diff
@@ -65,7 +70,12 @@ export class SbbTooltip implements ComponentInterface {
   /**
    * The state of the tooltip.
    */
-  @State() private _state: 'closed' | 'opening' | 'opened' | 'closing' = 'closed';
+  private set _state(state: SbbTooltipState) {
```

Comment:
Yes, I still use this approach in this case because otherwise it would break the appearance of the component by incorrectly positioning the tooltip arrow.